src/main/java/org/reaktivity/nukleus/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -133,6 +140,16 @@
 {
     private static final OctetsFW EMPTY_OCTETS = new OctetsFW().wrap(new UnsafeBuffer(new byte[0]), 0, 0);
 
+    private static final JsonBuilderFactory JSON = Json.createBuilderFactory(new HashMap<>());
+    private static final JsonObjectBuilder OBJECT_BUILDER = JSON.createObjectBuilder();
```

Comment:
These shared instances need to be non-static as they are mutable and not thread-safe.